Formula/snapcast.rb

```diff
@@ -0,0 +1,35 @@
+class Snapcast < Formula
+  desc "Synchronous multiroom audio player"
+  homepage "https://github.com/badaix/snapcast"
+  url "https://github.com/badaix/snapcast/archive/refs/tags/v0.26.0.tar.gz"
+  sha256 "166353267a5c461a3a0e7cbd05d78c4bfdaebeda078801df3b76820b54f27683"
+  license "GPL-3.0-only"
+
+  depends_on "boost" => :build
+  depends_on "cmake" => :build
```

Comment:
You also need `pkg-config`.

Reply:
Fixed in [529f17a](https://github.com/Homebrew/homebrew-core/pull/107089/commits/529f17a71e04f4c437904a716d72f0c9bd27c06f).